composer.json

```diff
@@ -9,28 +9,31 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.3",
```

Comment:
Why the lower version?

Reply:
As per the default laravel 7 repo, 7.2.5 is the required min version, so I updated to match this